site/src/docs/components/koros/accessibility.mdx

```diff
@@ -10,5 +10,7 @@ export default ({ children, pageContext }) => <TabsLayout pageContext={pageConte
 ## Accessibility
 
 ### Pay attention to
-- HDS Footer is designed to support many different brand colours. When customising colours, refer to <InternalLink href="/foundation/design-tokens/colour">colour guidelines</InternalLink> to ensure accessibility.
-- **When designing the footer link hierarchy, always think about keyboard and screen reader users.** Remember that keyboard users need to manually navigate to each element.
+
+- HDS Koros should be centered and have at least 2 full wave motif tips visible
+- Do not use Koro shapes as an accent element (not for example, for active element)
+- It's recomended to stick one with one Koro size and shape though the whole service
```

Comment:
I'd rephrase maybe a bit -> ...recommended to use the same Koro size and shape... though -> throughout (or across etc.)